webrtc.html

```diff
@@ -5024,10 +5024,71 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     </table>
   </section>
 
-  <section>
+  <section class="informative">
     <h2>Security Considerations</h2>
+    <p>This section is non-normative; it specifies no new behaviour,
+    but instead summarizes information already present in other parts
+    of the specification.</p>
+
+    <p>This document extends the Web platform with the ability to set
+    up real time, direct communication between browsers and other
+    devices, including other browsers.</p>
+
+    <p>This means that data and media can be shared between
+    applications running in different browsers, or between an
+    application running in a browser and something that is not a
+    browser, something that is an extension to the usual barriers in
+    the Web model against sending data between entities with different
+    origins.</p>
+
+    <p>The WebRTC specification provides no user prompts or chrome
+    indicators for communication; it assumes that once the Web page
+    has been allowed to access media, it is free to share that media
+    with other entities as it chooses.</p>
+
+    <p>A mechanism, <code>peerIdentity</code>, is provided that gives
+    Javascript the option of requesting media that the Javascript
+    cannot access, but can only be sent to certain other entities.</p>
+
+    <p>While a web application already before WebRTC will know the IP
+    address to which the application is delivered, setting up
+    communications exposes additional information about the browser’s
+    network context to the web application, including the set of IP
+    addresses available to the browser for WebRTC use. Some of this
+    information has to be passed to the corresponding party to enable
+    the establishment of a communication session.</p>
+
+    <p>Revealing IP addresses can leak location and means of
```

Comment:
I think this should be clearer about the distinction between the counterparty and the server.

As you say, the server knows the public IP but not the private IP (including IPs between VPNs).

The counterparty knows none of these...